Player.py

```diff
@@ -22,6 +22,13 @@ def __init__(self, x, y, playerId):
         self.y = y
         self.speed = 3
         self.state = 0
+        self.everyPlayersBombs = []
+        self.bombPlantedThisRound = 0
+        self.myBombsTimers = []
```

Comment:
Either "player's" something or "my" something, choose one. I would choose "player's"

Reply:
and I think that `playersBombs` would be the best, instead of `everyPlayersBombs`